src/js/videojs.wavesurfer.js

```diff
@@ -214,7 +214,7 @@ class Wavesurfer extends Plugin {
                 this.player.loadingSpinner.show();
 
                 // start loading file
-                this.load(options.src);
+                this.load(options.src, options.peaks);
```

Comment:
if you're going to add a new option, can you also add it to the documentation (and default settings)?